src/xb-silo.h

```diff
@@ -1,4 +1,6 @@
-/*
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: t; c-basic-offset: 8 -*-
+ * vi:set noexpandtab tabstop=8 shiftwidth=8:
```

Comment:
Could we just add a .editorconfig instead? https://editorconfig.org/

Reply:
Done